R/accounts.R

```diff
@@ -60,74 +60,121 @@ connectApiUser <- function(account = NULL, server = NULL, apiKey, quiet = FALSE)
 
   registerAccount(
     serverName = server,
-    accountName = account %||% user$username,
+    accountName = user$username,
```

Comment:
This change means that the `account` argument is unnecessary going forward. Something for the list.

Reply:
Hmmm, maybe I should leave as is for now. If we fix, I think it should be part of a general re-thinking of these functions (e.g. more consistent function names, argument names, and defaults)